Update Info about community calls (#328)
SAP/cloud-sdk: docs/community/community-call.mdx

## Community Call

> In this series of calls for SAP Community, we will give you regular updates on the latest releases of the SAP Cloud SDK, its roadmap and topics around extensibility of SAP Business Suite, SAP S/4HANA, and other SAP offerings using SAP Business Technology Platform.

Reviewer: There is a warning from vale for this sentence. If I am honest I don't fully understand the second half of this sentence - is it about providing the roadmap for S/4HANA?
Please convert this to an unordered list as this is the recommended way to treat lists.
I added the in-depth part from below and actually removed the last sentence, because in my eyes it was just empty marketing talk ;) Feel free to adjust as you like.
```suggestion
In this series of calls for SAP Community you will learn about:
- the latest releases of the SAP Cloud SDK
- in-depth focus topics around the SDK
- the SDK's roadmap
```

> The call series will be conducted by the SAP Cloud SDK development and product management team.

Reviewer: Use present tense
```suggestion
The call series is conducted by the SAP Cloud SDK development and product management team.
```

> We will introduce the SAP Cloud SDK and cover its latest updates along with a deep dive on the topic chosen for the day.

Reviewer: We don't really "introduce" the SDK, those are rather updates for people who already know the SDK on a high level.
The fact that we cover the latest updates was already mentioned above. I would remove this sentence and add it to the list above.

Author: Yes, you are right, removing this line.

The community update calls take place every second Tuesday of a given month.

> On this page, you can find the schedule as well as recordings of past sessions below.

Reviewer: ```suggestion
On this page, you can find the schedule as well as recordings of past sessions.
```

or

```suggestion
On this page, you can find the schedule as well as recordings of past sessions (see below).
```

[Register for the series.](https://sap-se.zoom.us/webinar/register/WN_dBtYmF5IR7Gb8TMjHqEvhw)

> We are working on enabling registering for individual sessions instead of a series, we will keep you posted on the progress.

Reviewer: ```suggestion
We are working on enabling registration for individual sessions instead of a series, we will keep you posted on the progress.
```

## Community Call

In this series of calls for SAP Community you will learn about:
- The latest releases of the SAP Cloud SDK
- An In-depth focus topic around the SDK

> - The SDK's roadmap

Reviewer 2: @artemkovalyov @KavithaSiva something is off with our build pipeline I think. These changes should cause the pipeline to fail, because the Value check should fail, because `The SDK` is not allowed. It must be `the SAP Cloud SDK` instead.

In fact it causes subsequent PR builds to fail, but I am clueless why Vale didn't pick up on this already in this PR. 
@KavithaSiva could you fix it in a follow up PR?
